modules/hosted-control-planes-overview.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * architecture/control-plane.adoc
+
+
+:_content-type: CONCEPT
+[id="hosted-control-planes-overview_{context}"]
+= Hosted control planes overview (technology preview)
+
+Hosted control planes for {product-title} provides a way to decouple your control plane from your worker nodes so that you can work in a hybrid cloud approach. You can enable hosted control planes through the multicluster engine for Kubernetes operator version 2.0. In that version, only Amazon Web Services (AWS) hosted clusters are available.
+
+:FeatureName: Hosted control planes
+include::snippets/technology-preview.adoc[]
+
+== Hosted control planes architecture
+
+Typically in {product-title}, a cluster consists of a control plane and a data plane. The control plane includes an API endpoint, a storage endpoint, a workload scheduler, and an actuator that ensures state. The data plane is the hot path, including compute, storage, and networking where workloads and applications run.
+
+The control plane and the data plane are coupled in terms of locality. The control plane is hosted by a dedicated group of nodes, which can be physical or virtual, with a minimum number to ensure quorum. The network stack is shared, and admin access to a cluster offers visibility into the cluster's control plane, machine management APIs, and other system components that contribute to the state of a cluster.
+
+While the coupled model works well, some situations require an architectural model where the control plane and data plane ar decoupled. In the decoupled model, the data plane is on a separate network domain with a dedicated physical hosting environment. The control plane is hosted by using high-level primitives such as deployments and stateful sets that are native to Kubernetes.
+
+image::hosted-control-planes-diagram.png[Diagram that compares the hosted control plane model against OpenShift with a coupled control plane and workers]
+
+== Benefits of hosted control planes
+
+Hosted control planes for {product-title} paves the way for a hybrid-cloud approach and offers several other benefits.
+
+The management plane for control planes and cloud credentials is separate from the cluster that you interact with. Because the control plane is managed, you are less likely to accidentally delete your clusters.
+
+When your control plane and data plane are coupled, you might host one control plane on three machines. With hosted control planes, you can host several control planes on three machines, and you can run hundreds of control planes on 150 nodes. As a result, high availability (HA) clusters are more affordable.
+
+Because the control plane consists of pods that are launched on {product-title}, you do not need to wait to provision machines. In addition, you gain Kubernetes features:
```

Comment:
```suggestion
Because the control plane consists of pods that are launched on {product-title}, control-plane bootstrap time is minimal. In addition, because the control-plane is just another Kubernetes workload, the same principles apply (e.g., monitoring, logging, autoscaling, etc).
```

Reply:
the problem with less or faster is that it compares, I try to avoid comparing if I can but It should workas well as it's a fact :)

Reply:
I rewrote the phrase to avoid making a comparison:

>Because the control planes consist of pods that are launched on OpenShift Container Platform, control planes start quickly.